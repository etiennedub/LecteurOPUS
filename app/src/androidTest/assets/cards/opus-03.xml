<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Opus>
  <ID>0000000000000004007100000000000092C192BD02500A44001FFB3B009000</ID>
  <Expiration>04789200091BA7AA8222000000000000000000000000000000000000009000</Expiration>
  <Transit>
    <List index="0">617DA800000CF000280328008087B0BED40600000000000000000000009000</List>
    <List index="1">61795800000CF000280328008087B0BCAC0600000000000000000000009000</List>
    <List index="2">61755C80000CF000280328008087B0BAAE4600000000000000000000009000</List>
  </Transit>
  <Subscription>
    <List index="0">1E0A0043B044C207F48D8227B6000000000020000000000000000000009000</List>
    <List index="1">1E0A0043B082C2FFF48D8428CA000000000020000000000000000000009000</List>
    <List index="2">1E0A092B80000007F48D79A6D2000000000020000000000000000000009000</List>
  </Subscription>
  <Ticket>
    <List index="0">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
    <List index="1">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
    <List index="2">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
  </Ticket>
</Opus>
