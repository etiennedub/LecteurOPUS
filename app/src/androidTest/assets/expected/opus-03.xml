<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OPUS</Type><ID>2462159549</ID><ExpirationDate>30/06/2017</ExpirationDate><Trip><List index="0" Date="31/01/2014" Time="14:08" Transfert="false" OperatorId="20" BusId="16" OperatorName="RTC" BusName="LEBUS 28" /><List index="1" Date="30/01/2014" Time="11:28" Transfert="false" OperatorId="20" BusId="16" OperatorName="RTC" BusName="LEBUS 28" /><List index="2" Date="29/01/2014" Time="11:37" Transfert="false" OperatorId="20" BusId="16" OperatorName="RTC" BusName="LEBUS 28" /></Trip><Contract><List index="0" Subscription="true" Valid="false" NbTicket="0" OperatorId="20" ValidityDate="31/12/2013" /><List index="1" Subscription="true" Valid="false" NbTicket="0" OperatorId="20" ValidityDate="31/01/2014" /><List index="2" Subscription="false" Valid="false" NbTicket="0" OperatorId="20" /></Contract></Card>